### PR TITLE
Copy Python tests for Modulemd.ModuleStream.rpm_artifacts into C

### PR DESCRIPTION
**Pull request by srikavin**
This is related to #199.

### CODE REVIEW

File: modulemd/tests/test-modulemd-modulestream.c [hunk 1]
```diff
@@ -206,6 +206,59 @@ module_stream_v2_test_profiles (ModuleStreamFixture *fixture,
...
+{
+  g_autoptr (ModulemdModuleStreamV1) stream = NULL;
+  g_auto (GStrv) artifacts = NULL;
+
+  stream = modulemd_module_stream_v1_new (NULL, NULL);
+
+  modulemd_module_stream_v1_add_rpm_artifact (
+    stream, "bar-0:1.23-1.module_deadbeef.x86_64");
+  artifacts = modulemd_module_stream_v1_get_rpm_artifacts_as_strv (stream);
+  g_assert_true (g_strv_contains ((const gchar *const *)artifacts,
```
Comment (OrionStar25) [inline modulemd/tests/test-modulemd-modulestream.c]: Please also check that length of this is equal to 1.

File: modulemd/tests/test-modulemd-modulestream.c [hunk 2]
```diff
@@ -206,6 +206,59 @@ module_stream_v2_test_profiles (ModuleStreamFixture *fixture,
...
+{
+  g_autoptr (ModulemdModuleStreamV2) stream = NULL;
+  g_auto (GStrv) artifacts = NULL;
+
+  stream = modulemd_module_stream_v2_new (NULL, NULL);
+
+  modulemd_module_stream_v2_add_rpm_artifact (
+    stream, "bar-0:1.23-1.module_deadbeef.x86_64");
+  artifacts = modulemd_module_stream_v2_get_rpm_artifacts_as_strv (stream);
+  g_assert_true (g_strv_contains ((const gchar *const *)artifacts,
```
Comment (OrionStar25) [inline modulemd/tests/test-modulemd-modulestream.c]: Same as above

File: modulemd/tests/test-modulemd-modulestream.c (hunk 1 above)
Comment (srikavin) [reply modulemd/tests/test-modulemd-modulestream.c]: I have added this check.

**Comment by srikavin** [comment]
@sgallagher I've rebased my branch onto master.

**Pull request merged**